docs/source/author/topics.rst

```diff
@@ -767,6 +767,11 @@ Programming Exercises Configuration File
 
       - ``learning-outcomes:`` A list of keys corresponding to learning outcomes.
 
+      - ``extra-challenge:`` A Markdown filename containing the content for an
+        extra challenge.
+        The Markdown file should be within the programming exercise folder, and
+        must not include a heading (should only contain text).
```

Comment:
The first line is enough for this section, details about where to save the md file and what should be in it should be moved to section for 'Adding Programming Exercises'